pkg/config/config.go

```diff
@@ -71,6 +71,77 @@ type S3AuthInfo struct {
 	}
 }
 
+// Database - holds metadata KV configuration
```

Comment:
This change is unrelated to the PR.  Please revert it and perform is in another PR.

Reply:
Of course it is related to this PR.
I'm reusing the Database struct for the ACL server config.
If you want me to break this into pieces and first merge this change I can do that. But this is required for this PR